force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +97,18 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
+       List<String> words = sentence.split(' ', sentence.length());
 
         // Create a set to store the unique words
+        Set<String> uniqueWords = new Set<String>(words);
 
         // Add each word to the set
 
         // Join the unique words back into a sentence
+       String rejoinedSentence = String.join(new List<String>(uniqueWords), ' ');
 
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return rejoinedSentence; // Replace null with the variable you used to store the result
```

Comment:
pretty good! The only thing I did differently was  add the List directly to the the Set. 

```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```